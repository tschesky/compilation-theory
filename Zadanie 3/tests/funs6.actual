Visiting program
Used undefined symbol res in line 8
Type checking finished
