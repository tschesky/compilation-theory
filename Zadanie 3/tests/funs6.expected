Error: Function identifier 'pow' used as a variable: line 6
Error: Function identifier 'pow' used as a variable: line 8
Error: Variable 'res' undefined in current scope: line 8
Error: Function identifier 'pow' used as a variable: line 11
